listenbrainz/webserver/templates/footer.html

```diff
@@ -0,0 +1,323 @@
+<section class="footer">
+    <div class="container-fluid">
+        <div class="row mb-4 mt-2 pt-2">
+            <div class="col-md-6">
+                <h3 class="ms-3">
+                    <img src="{{ url_for('static', filename='img/listenbrainz-logo.svg') }}" width="196" alt="ListenBrainz"/>
+                </h3>
+                <br />
+                <p class="ms-3">
+                    You can use ListenBrainz to track your listening habits, discover new music with personalized recommendations, and share your musical taste with others using our visualizations.
+                </p>
+                <ul class="list-unstyled ms-3">
+                    <li>
+                <span>
+                  Development IRC:
+                </span>
+                        <a
+                                class="fw-bold ms-1"
+                                href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz"
+                                rel="noopener noreferrer"
+                                target="_blank"
+                        >
+                            #metabrainz
+                        </a>
+                    </li>
+                <span>
+                  Forums:
+                </span>
+                        <a
+                                class="fw-bold  ms-1"
+                                href="https://community.metabrainz.org"
+                                rel="noreferrer"
+                                target="_blank"
+                        >
+                            community.metabrainz.org
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="col-sm-12 col-md-3">
+                <h3 class="ms-3">
+                    Fellow Projects
+                </h3>
+                <ul class="list-style-type">
+                    <li class="pt-2 pb-2">
```

Comment:
I think there's an error here, there's no MusicBrainz but ListenBrainz is here which we don't need.
I also think that means that maybe (?) you can remove the `meb-icons/ListenBrainz.svg` file you added in this PR (if it's only used in the footer)

Reply:
Thanks! Fixed.